### ISSUE TITLE
Docs: Not clear that development versions are slower

### ISSUE THREAD

**Issue by scarlac** [facebook/hermes#97]
This is a general feedback / documentation ticket.

My team enabled Hermes for an existing app and we didn't notice any immediate functionality issues. We did however notice that the entire app felt slow. But I initially brushed it off as a glitch.
A tester asked me about it so to verify our suspicion we video recorded a build with and one without Hermes. The difference was staggering, around ~3-5x the time.

I fear that other teams that enable it may not immediately notice as many devs do iOS-first development. While production builds _may_ be faster, development is significantly slowed down. Had it been a 1.5x increase, I would not mind. But up to around ~5x is problematic.

After checking the [blog post about it](https://engineering.fb.com/android/hermes/), it's not clear that this is the case. The section about it is called "Lazy compilation" and the word "slow" or any "negative" word is not used:
> To keep reloads fast, Hermes debug builds don’t use ahead-of-time compilation; instead, they generate bytecode lazily on device [...] The trade-off is that lazy-compiled bytecode does not include all the optimizations of a production build

Instead of just putting it in plain english, the reader is left to figure out that 'not include all the optimizations' means 'will run significantly slower'.

The reason I focus on the blog post is because the website does not mention this caveat. Neither does the GitHub page. And neither does the React Native documentation.

**My suggestion** would simply to be up front about it: Hermes speeds up production builds and slows down development builds.
We shouldn't be ashamed of admitting to that, it's just another engineering trade-off.

**Comment by tmikov** [comment]
This is caused by https://github.com/facebook/hermes/issues/73 . Once that is resolved (it is simply a matter of reconfiguring the build), the dev versions should be fast.

**Comment by scarlac** [comment]
You can close this ticket, @tmikov, if it'll be tracked in #73
